packages/nvidia_nat_harbor/tests/test_library_mode_contracts.py

```
@@ -0,0 +1,92 @@
+from nat_harbor.agents.installed.library_mode import NemoInlineRunnerResult
+from nat_harbor.verifier.inline_verifier import InlineVerifierRequest
+from nat_harbor.verifier.inline_verifier import InlineVerifierResult
+from nat_harbor.verifier.inline_verifier import build_inline_verifier_metadata
+
+
+def test_nemo_inline_runner_contracts_roundtrip(tmp_path: Path) -> None:
+    artifact_dir = tmp_path / "agent"
+    result_path = artifact_dir / "trajectory.json"
+    request = NemoInlineRunnerInput(
+        instruction="hello",
+        config_file="/tmp/config.yml",
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/config.yml"

(S108)